tests/unit/s2n_stuffer_network_order_test.c

```diff
@@ -21,6 +21,8 @@
 
 #include "stuffer/s2n_stuffer_network_order.c"
```

Comment:
Is there a way around this? There is only 1 static function in the network_order file.